### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by jane-gorlova**
Updated the README.md to describe each input

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,151 @@
-# Run Jupyter Notebook action for neuro-flow to be used with Neu.ro template
+# Run a Jupyter Notebook instance
 
-This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/). It's intended to be used with Neu.ro [platform template](https://github.com/neuro-inc/cookiecutter-neuro-project), but can be adapted for other use cases as well.
+This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching a [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/). It's intended to be used with the Neu.ro [platform template](https://github.com/neuro-inc/cookiecutter-neuro-project), but can be adapted for other use cases as well.
```
Comment (YevheniiSemendiak) [inline README.md]: Let's also mention Jupyter Lab.

```suggestion
This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching a Jupyter [Notebook](https://jupyter-notebook.readthedocs.io/en/stable/) or [Lab](https://jupyterlab.readthedocs.io/en/stable/). It's intended to be used with the Neu.ro [platform template](https://github.com/neuro-inc/cookiecutter-neuro-project), but can be adapted for other use cases as well.
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,151 @@
...
 
-It requires image name to run Notebook on as well as references to 5 volumes: data, code, config, notebooks and results, which will be mounted to `/project/data`, `/project/modules`, `/project/config`, `/project/notebooks` and `/project/results` respectively.
+It requires the name of the image on which to run Notebook, as well as references to 5 volumes: data, code, config, notebooks and results. These volumes will be mounted to `/project/data`, `/project/modules`, `/project/config`, `/project/notebooks`, and `/project/results` respectively.
 
-After launch web UI open up in default browser automatically.
\ No newline at end of file
+After the Notebook instance is launched, its Web UI will be automatically opened in the default browser.
```
Comment (YevheniiSemendiak) [inline README.md]:
```suggestion
After the Jupyter instance is launched, its Web UI will be automatically opened in the default browser.
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,151 @@
...
+
+### Quick example:
+
+```
+jobs:
+  jupyter:
+    action: gh:neuro-actions/jupyter@master
+    args:
+      image: neuromation/base
```
Comment (YevheniiSemendiak) [inline README.md]: Let's set `neuromation/base:latest` as a default image used by this action (and update the action description respectively).
WDYT?

File: README.md [hunk 4]
```diff
@@ -1,7 +1,151 @@
...
+The name of the image on which to run the Notebook instance.
+
+### Example
+
+```
+args:
+    image: neuromation/base
+```
+
+### `preset`
```
Comment (YevheniiSemendiak) [inline README.md]: Let's set a default value for the `preset` input equal to `""` (in this case, the default preset will be used, if the action arg is not provided).

File: README.md [hunk 5]
```diff
@@ -1,7 +1,151 @@
...
+Whether to use HTTP authentication for Jupyter or not. `"True"` by default.
+
+### Example
+
+```
+args:
+    http_auth: "False"
+```
+
+### `volumes_data_remote`
```
Comment (YevheniiSemendiak) [inline README.md]: I would suggest putting all inputs without a default value at the top of the argument list.

**Review by YevheniiSemendiak — COMMENTED** [review]
Several suggestions, left, might be relevant to improve the action.
 I might also submit them as the PR into your PR if you would like.
Pls, ping me if needed.

File: README.md (hunk 5 above)
Comment (jane-gorlova) [reply README.md]: Addressed in a commit.

File: README.md (hunk 1 above)
Comment (jane-gorlova) [reply README.md]: Addressed in a commit

Comment (YevheniiSemendiak) [reply README.md]: awesome, thank you!

File: README.md (hunk 5 above)
Comment (YevheniiSemendiak) [reply README.md]: great! I will proceed with other changes in action.yaml and we are good :)

**Review by YevheniiSemendiak — APPROVED** [review]
Looks good to me! Thank you! 👍

**Pull request merged**